stock_route_mto/hooks.py

```diff
@@ -0,0 +1,8 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+# Copyright 2025 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+
+
+def post_init_hook(env):
+    warehouses = env["stock.warehouse"].browse()
```

Comment:
This code doesn't do anything, you wanted to get all WH no?
```suggestion
    warehouses = env["stock.warehouse"].search([])
```

Reply:
fixed